src/System.ComponentModel.Composition/src/System/ComponentModel/Composition/Hosting/CatalogExportProvider.cs

```diff
@@ -940,7 +940,7 @@ private void EnsureCanSet<T>(T currentValue)
         {
             if ((_isRunning) || (currentValue != null))
             {
-                throw new InvalidOperationException(string.Format(CultureInfo.CurrentCulture, SR.ObjectAlreadyInitialized));
+                throw new InvalidOperationException(SR.ObjectAlreadyInitialized);
```

Comment:
Why can we remove `CultureInfo.CurrentCulture` here but not in other places like line 912 above? Is it because its a regular string with no parameters to interpolate?

https://github.com/dotnet/corefx/blob/c390ce7df50252e11f5d322276e9d19e046d1332/src/System.ComponentModel.Composition/src/Resources/Strings.resx#L150-L152

vs 

https://github.com/dotnet/corefx/blob/c390ce7df50252e11f5d322276e9d19e046d1332/src/System.ComponentModel.Composition/src/Resources/Strings.resx#L153-L155

Reply:
> Why can we remove CultureInfo.CurrentCulture here but not in other places like line 912 above?

It's not just the provider that's being removed here... it's the whole SR.Format, since there's nothing to format.

That said, I'm pushing a commit that also removes the CurrentCulture from everywhere that uses it, since it's the default.